content/backend/graphql-go/6-authentication.md

```diff
@@ -236,6 +236,7 @@ func ForContext(ctx context.Context) *users.User {
 </Instruction>
 
 Now we can use the middleware we created in our server:
+Create a new folder in the root directory called 'server' and move the server.go file to the directory you just created.
```

Comment:
This is a good suggestion which I did not know about when I wrote this. I also prefer to call the directory `cmd` as it's [more common](https://github.com/golang-standards/project-layout#cmd) in the Go community.